cleo/registry.py

```diff
@@ -104,6 +118,10 @@ def connect_light_to_ldd_for_ng(
         i_source = self.subgroup_idx_for_light[light]
         light_prop_syn.epsilon[i_source, :] = epsilon
         light_prop_syn.T[i_source, :] = light.transmittance(coords_from_ng(ng)).ravel()
+        light_prop_syn.scan_period = second / light.scan_freq
```

Comment:
I'd put these in a separate function that you call both here (on injection) and when you change the params on the scope/imaging light (setter functions). like `update_raster_scan_params(scope)` or something like that

Reply:
like you've already done with `update_fov` but can include these other variables too